tests/src/Interop/PInvoke/Attributes/LCID/LCIDTest.cs

```diff
@@ -0,0 +1,131 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+//The testcase focus test the BStr with embed null string
+using System.Runtime.InteropServices;
+using System;
+using System.Reflection;
+using System.Text;
+using CoreFXTestLibrary;
+
+class LCIDTest
+{
+    [DllImport(@"LCIDNative.dll", EntryPoint = "Marshal_InOut1")]
+    [LCIDConversionAttribute(0)]
+    [return: MarshalAs(UnmanagedType.LPStr)]
+    private static extern StringBuilder MarshalStrB_InOut1([In, Out][MarshalAs(UnmanagedType.LPStr)]StringBuilder s);
```

Comment:
nit: why entrypoint name(Marshal_InOut1) and function name(MarshalStrB_InOut1) is different? I would suggest keep these two same name unless you were testing EntryPoint.

Reply:
Fixed